lib/views/message.coffee

```diff
@@ -1,14 +1,14 @@
 # This file is imported in views/panel
 
 class Message extends HTMLElement
-  initialize: (@message, @addPath) ->
+  initialize: (@message, @options) ->
 
   attachedCallback: ->
     @appendChild Message.renderRibbon(@message)
-    @appendChild Message.renderMessage(@message)
-    @appendChild Message.renderLink(@message, @addPath) if @message.filePath
+    @appendChild Message.renderMessage(@message, @options)
+    @appendChild Message.renderLink(@message, @options) if @message.filePath
 
-  @renderLink: (message, addPath) ->
+  @renderLink: (message, {addPath}) ->
```

Comment:
I would respectfully disagree that "shouldn't have" applies here. Usually with options-object, it's considered a "good practice" to have a sensible default. I would assume here that would be `false`, since at least one is `false` in 3 out of 4 instances.